crypto/pem_cert_pool.go

```diff
@@ -0,0 +1,70 @@
+package crypto
+
+import (
+	"encoding/pem"
+
+	"github.com/golang/glog"
+	"github.com/google/certificate-transparency/go/x509"
```

Comment:
BTW not actually sure we can use the golang x509 as one of the changes was to make TBSCertificate visible.